lib/sass/calcite-web/components/_alert.scss

```diff
@@ -6,76 +6,106 @@
 
 @mixin alert() {
   @include font-size(-1);
-  padding: $baseline/2 $baseline/2;
   @include text-color($transparent-black);
-  background-color: $lightest-blue;
-  position: relative;
   display: none;
+  position: relative;
+  border-top: 3px solid;
   z-index: 100;
   max-width: 40em;
-  border: 1px solid $blue;
-  @include box-shadow($box-shadow);
-  @include link-color($off-black, $black);
+  border-radius: 2px 2px 0 0;
+  box-shadow: 0 0 12px 0 rgba($black, 0.15);
   &.is-active {
-    display: block;
-  }
-  a:hover {
-    color: $transparent-black;
+    display: flex;
   }
 }
 
-@mixin alert-red() {
-  background-color: $lightest-red;
-  border-color: $light-red;
+@mixin alert-element-base() {
+  padding: $baseline/2 $baseline/2;
+  flex: 0 0 auto;
+  & svg {
+    position: relative;
+    vertical-align: top;
+    height: 16px;
+    width: 16px;
+  }
 }
 
-@mixin alert-yellow() {
-  background-color: $lightest-yellow;
-  border-color: $yellow;
+@mixin alert-icon() {
+  @include alert-element-base;
+  display: flex;
+  align-items: center;
 }
 
-@mixin alert-green() {
-  background-color: $lightest-green;
-  border-color: $light-green;
+@mixin alert-content() {
+  @include alert-element-base;
+  flex: 1 1 auto;
```

Comment:
we should throw a `min-width: 0` and probably a `word-wrap: break-word` on here so the layout will work with long, unbroken strings:

![Screen Shot 2019-03-21 at 4 33 04 PM](https://user-images.githubusercontent.com/1031758/54791440-21590c00-4bf7-11e9-9e64-76bbc0813c31.png)

Reply:
Good catch, added.